cypress/integration/folder-contents.js

```diff
@@ -108,15 +102,33 @@ if (Cypress.env('API') !== 'guillotina') {
         path: 'my-folder',
       });
       cy.visit('my-folder/contents');
-      cy.get('.icon.configuration-svg').click();
+      cy.get('.icon.configuration-svg').click({ force: true });
       cy.get('.sort_CreationDate').invoke('trigger', 'mouseover');
       cy.get('.item.sort_created_descending').click({ force: true });
-
-      // then the last document created must be the firs element
-      cy.get('tbody>tr:nth-child(1) .expire-align > span').should(
-        'have.text',
-        ' My Child',
+      cy.waitForResourceToLoad(
+        'my-folder/@search?path.depth=1&sort_on=created&sort_order=descending',
       );
+      cy.wait(5000);
```

Comment:
@ksuess can you elaborate on why waitForResourceToLoad is not sufficient here? Wait statements slow down the test runs and should be avoided if possible.